recipes/default.rb

```diff
@@ -407,3 +407,20 @@
   group node['hops']['group']
   mode 500
 end
+
+if not node['hops']['aws_access_key_id'].eql?("") and not node['hops']['aws_secret_access_key'].eql?("")
+  directory "/home/#{node['hops']['hdfs']['user']}/.aws" do
```

Comment:
Use this method to assemble the home directory of the user: https://github.com/logicalclocks/conda-chef/blob/60639d26a5644dd65b462607a038e8267c80d557/libraries/default.rb#L14